app/src/main/java/com/artemzin/qualitymatters/ApplicationModule.java

```diff
@@ -33,6 +34,11 @@ public QualityMattersApp provideQualityMattersApp() {
     }
 
     @Provides @NonNull @Singleton
+    public Context provideContext() {
```

Comment:
So let's just accept `app` as parameter, I'm too scared to provide `Context` globally 😅